<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>581</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnSetDefault">
        <property name="text">
         <string>Set Default</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="SettingsTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="GeneralSettingsWidget">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>521</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="GeneralSettingsLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="rootDirectoryLabel">
           <property name="text">
            <string>Root Directory:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QPushButton" name="btnRootDirectory">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="rootDirectoryLineEdit"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="logDirectoryLabel">
           <property name="text">
            <string>Log Directory: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QPushButton" name="btnLogDirectory">
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="logDirectoryLineEdit"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="logLevelLabel">
           <property name="text">
            <string>Log Level</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="logLevelComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="cloudProviderLabel">
           <property name="text">
            <string>Cloud Provider</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cloudProviderComboBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="themeLabel">
           <property name="text">
            <string>Theme</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="themeComboBox"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="InputSettingsWidget">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="InputSettingsLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="headerCaseLabel">
           <property name="text">
            <string>Header Case</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="headerCaseComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="headerSpacesLabel">
           <property name="text">
            <string>Header Spaces</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="headerSpacesComboBox"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="outputSettings">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>421</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="outputSettingsFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="separatorLabel">
           <property name="text">
            <string>Separator</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="separatorComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="encodingLabel">
           <property name="text">
            <string>Encoding</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="encodingComboBox"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
